bee-ledger/src/workers/snapshot/error.rs

```diff
@@ -13,6 +13,9 @@ pub enum Error {
     /// I/O error happened.
     #[error("I/O error happened: {0}")]
     Io(#[from] std::io::Error),
+    /// Types error.
+    #[error("Types error: {0}")]
```

Comment:
Can you lowercase all errors ?

Reply:
I'll also add dots. 😁

Reply:
I think convention is also not to end an error message with punctuation (I hope you haven't added all those periods yet...😅)